src/Vault.sol

```diff
@@ -378,6 +396,8 @@ contract Vault {
      */
     function liquidateVault(address liquidationKeeper) public onlyFactory returns (bool success, address liquidator_) {
         //gas: 35 gas cheaper to not take debt into memory
+        (address[] memory assetAddresses, uint256[] memory assetIds, uint256[] memory assetAmounts) =
```

Comment:
Variables not used

Reply:
ah yes,  this one is not necessary anymore